src/extension/php_objects.c

```diff
@@ -14,6 +14,15 @@ static zend_llist _function_entry_arrays;
 
 static void _unregister_functions(void *zfe_arr_vp);
 
+typedef struct _dd_registered_entries {
+    char *name;
+    char *env_name;
```

Comment:
I assume these two variables are not nullable, if so use the `nonnull` attribute, otherwise use `nullable`.

Reply:
Done